superdesk/content_types_async/module.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.core.resources import ResourceConfig, MongoResourceConfig, MongoIndexOptions
+from superdesk.types.content_types import ContentTypes
+from .service import ContentTypesService
+
+
+content_types_resource_config = ResourceConfig(
+    name="content_types",
+    data_class=ContentTypes,
+    service=ContentTypesService,
+    mongo=MongoResourceConfig(
```

Comment:
Missing `rest_endpoints` config, otherwise no REST API will be exposed for this resource. Also missing the `default_sort` config

```python
from superdesk.core.resources import RestEndpointConfig
from superdesk.core.auth.privilege_rules import http_method_privilege_based_rules

CONTENT_TYPE_PRIVILEGE = "content_type"

content_types_resource_config = ResourceConfig(
    ...
    default_sort=[("priority", -1)],
    rest_endpoints=RestEndpointConfig(
        id_param_type=r'regex("[\w,.:-]+")',
        auth=http_method_privilege_based_rules(
            {
                "POST": CONTENT_TYPE_PRIVILEGE,
                "PATCH": CONTENT_TYPE_PRIVILEGE,
                "DELETE": CONTENT_TYPE_PRIVILEGE,
            }
        ),
    ),
)
```